<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntroWindow</class>
 <widget class="QMainWindow" name="IntroWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>1000</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>키오스크</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="goBackButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>870</y>
      <width>700</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string>뒤로가기</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>700</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string>주문하신 내역을 확인하세요.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="PayButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>770</y>
      <width>700</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string>결제하기</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="typeLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>700</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>34</pointsize>
     </font>
    </property>
    <property name="text">
     <string>포장</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="BreadButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>640</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>밀빵</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>src/images/assets/bread/whole_wheat_bread.png</normaloff>src/images/assets/bread/whole_wheat_bread.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="MeatButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>640</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>스테이크</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>src/images/assets/meat/steak.png</normaloff>src/images/assets/meat/steak.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="CheeseButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>640</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>모짜렐라</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>src/images/assets/cheese/mozzarella.png</normaloff>src/images/assets/cheese/mozzarella.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="VegatableButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>640</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>채소 변경하기</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="SauceButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>640</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>소스 변경하기</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="BeverageButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>640</y>
      <width>640</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>사이다</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>src/images/assets/beverage/cider.png</normaloff>src/images/assets/beverage/cider.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
